src/Components/Header.jsx

```diff
@@ -82,6 +82,12 @@ function Header() {
     searchMovies(searchInput.current.value);
   };
 
+  const handleKeyDown = (e) => {
+    if (e.key === "Enter") {
+      searchMovies(searchInput.current.value);
+    }
+  };
+
   const navigate = useNavigate();
```

Comment:
You could take the opportunity to move this const to the top, (line 73)

Reply:
OK!